src/tests/ftest/container/create.py

```diff
@@ -43,7 +43,7 @@ def test_container_create(self):
 
         Test Description: valid and invalid container creation and close.
 
-        :avocado: tags=all,container,tiny,smoke,full_regression,containercreate
+        :avocado: tags=all,container,pr,daily_regression,medium
```

Comment:
Is this PR really about making test bucket changes?

Reply:
Was using this test to experiment some things. Will revert the changes here in next patch after rebase.